src/common/connection.ts

```diff
@@ -8,7 +8,7 @@ import {RippledError, DisconnectedError, NotConnectedError,
   RippledNotInitializedError} from './errors'
 
 export interface ConnectionOptions {
-  trace?: boolean
+  trace?: boolean | ((id: string, msg: string) => void)
```

Comment:
super nit (feel free to ignore): I prefer to not use abbreviations in code. It makes things harder to reason about. 

I.E. if this is `msg`, why do we call it `proxyAuthorization` instead of `proxyAuth`? There's no clear rule about when to use or not use abbreviations, so it's easier to just never use them by default.

Reply:
Personally I feel the same way, although I think some ultra-common generic terms get a pass: `id`, `err`, `msg`, etc. are all fine by me.

I do see `message` used elsewhere in the codebase, so let's make the switch for consistency's sake, if nothing else.